pkg/inject/inject_test.go

```diff
@@ -0,0 +1,60 @@
+package inject
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type injectionTestCase struct {
+	description                         string
+	annotatedPodTemplateSpecPath        string
+	expectedInjectedPodTemplateSpecPath string
+}
+
+func TestSidecarInjection(t *testing.T) {
+	var testCases = []injectionTestCase{
+		{
+			description:                         "Secretless",
+			annotatedPodTemplateSpecPath:        "./fixtures/secretless-annotated-pod.json",
+			expectedInjectedPodTemplateSpecPath: "./fixtures/secretless-mutated-pod.json",
+		},
+		{
+			description:                         "Kubernetes Authenticator",
+			annotatedPodTemplateSpecPath:        "./fixtures/authenticator-annotated-pod.json",
+			expectedInjectedPodTemplateSpecPath: "./fixtures/authenticator-mutated-pod.json",
```

Comment:
I think you have to use `testdata` as the folder name to prevent golang trying to parse it: https://go.googlesource.com/go/+/2f9eeea212681927385a4713004438cd864966e0/src/cmd/go/test.go#62